pkg/simple/client/devops/pipeline.go

```diff
@@ -79,7 +79,7 @@ type Pipeline struct {
 	NumberOfFolders                int           `json:"numberOfFolders,omitempty" description:"number of folders"`
 	NumberOfPipelines              int           `json:"numberOfPipelines,omitempty" description:"number of pipelines"`
 	PipelineFolderNames            []interface{} `json:"pipelineFolderNames,omitempty" description:"pipeline folder names"`
-	WeatherScore                   int           `json:"weatherScore,omitempty" description:"the score to description the result of pipeline activity"`
```

Comment:
This may result in 500 if the field does not exist. Has it been fully tested?

Reply:
The front-end cannot see `WeatherScore` if its value is `0`.  Because it is ignored due to `omitempty` field.
An int field always has the default value `0`. So it always has a value. I have no idea which cases might lead to a 500 result.

Reply:
When it unmarshal JSON data that the field of WeatherScore does not exist, it will result in an error.

Reply:
I cannot produce the error case which you mentioned. But I try to unmarshal a JSON data which does not contain the field WeatherScore, nothing happened. You can run the demo code below:

https://play.golang.org/p/MJFcRRISY4i